nextjs-metrics-viewer/app/login/page.tsx

```
@@ -0,0 +1,161 @@
+import React, { useState, useEffect } from "react";
+import { useAuth } from "@/context/AuthContext";
+import { useRouter, useSearchParams } from "next/navigation";
+import { Alert, Button } from "flowbite-react";
+import { Icons } from "@/components/icons";
+import type { SVGProps } from "react";
+import type { OverridableComponent } from "@mui/material/OverridableComponent";
+import type { SvgIconTypeMap } from "@mui/material/SvgIcon";
+
+
+function createIconWrapper(
+  MUIIcon: OverridableComponent<SvgIconTypeMap<{}, "svg">>
```

Biome (1.9.4)

[error] Don't use '{}' as a type.

Prefer explicitly define the object shape. '{}' means "any non-nullable value".

(lint/complexity/noBannedTypes)